static/style.css

```diff
@@ -124,13 +127,41 @@ header h2 {
     border-top: 2px solid #262119ff;
     border-bottom: 2px solid #262119ff;
 }
+#hamburger {
+    display: none;
+}
+
 @media only screen and (max-width: 600px) {
-    nav ol {
+    #hamburger {
+        display:inline-block;
+        position: fixed;
+        top: 0px;
+        right: 2%;
+        z-index: 2;
+        padding: 4%;
+        border: solid 1px #53462bff;
+        border-radius: 30%;
+        font-size: 1.5rem;
+        font-weight: 800;
+        color: #53462bff;
+    }
+    #hamburger:hover {
+        cursor: pointer;
+    }
+    .navbar {
+        display: none;

```

Comment:
I'd add a comment here to explain that this gets toggled to visible when the user clicks the hamburger menu. 
```
display: none; /* clicking .hamburger menu changes this to block */
```